developer-docs/kerberos.md

```diff
@@ -162,9 +162,9 @@ Useful tooling on the instance includes:
 * [`pwsh`](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-6) - PowerShell 6
 * [`omicli`](https://github.com/microsoft/omi/blob/master/Unix/cli/examples.txt) - client tool used to verify basic OMI server functionality
 
-#### TravisCI automated testing
+#### GitHub Actions automated testing
 
-At present, Travis setup will:
```

Comment:
Grepping for Travis on this branch, it looks like this also needs to change in `developer-docs/releasing-bolt.md`.

Reply:
Also in doc/index.html and doc/file.README.html

Reply:
I'm not sure what `doc/index.html` and `doc/file.README.html` are?

Reply:
Oh, ha, they're leftover docs generated when I was testing a different PR. Nevermind!

Reply:
`developer-docs/releasing-bolt.md` references the Travis tests for another repo and shouldn't be changed.

Reply:
I'm not sure what `doc/index.html` and `doc/file.README.html` are?